README.md

```diff
@@ -23,7 +24,7 @@ latest: Pulling from nathantheinventor/open-contest
 Digest: sha256:9f65996f196f8780956cd08b9ed53d84f4e26c5e8456fe50c6487e8a5f316948
 Status: Image is up to date for nathantheinventor/open-contest:latest
 
-$> docker run -v /tmp:/tmp -v /home/user/db:/db -v /var/run/docker.sock:/var/run/docker.sock -p 0.0.0.0:
+$> docker run -v /tmp:/tmp -v ~/db:/db -v /var/run/docker.sock:/var/run/docker.sock -p 0.0.0.0:
```

Comment:
I had it this way because docker gives me an error if I try to run it with a relative path

Reply:
~/db expands to an absolute path (the shell replaces ~ with /home/user, or whatever the user's home directory is).

Reply:
I thought I had remembered trying this and it didn't work, but I tried it again and it worked now